packages/hash/docker/frontend/prod/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM node:16.16.0-alpine AS builder
```

Comment:
```suggestion
FROM node:16.18.1-alpine AS builder
```

(same could apply elsewhere)

Good that we're sticking with Node 16 because Node 18 is not yet supported in Vercel Lambdas:
<img width="929" alt="Screenshot 2022-11-14 at 15 53 06" src="https://user-images.githubusercontent.com/608862/201705157-178d750c-1799-47aa-afbe-f9bf55320bde.png">

Sticking with a single major version everywhere makes sense.

Reply:
Should I be incrementing the version in the other dockerfiles as well?

Reply:
Addressed in 9557bff473b416dd65444b5897fbbce1015b1b01 Thanks :)